packages/location-management/src/components/LocationUnitGroupAddEdit/index.tsx

```diff
@@ -14,18 +14,22 @@ export interface Props {
 }
 
 export const LocationUnitGroupAddEdit: React.FC<Props> = (props: Props) => {
+  const [title, setTitle] = useState('');
   const params: { id: string } = useParams();
   const { opensrpBaseURL } = props;
+
   return (
     <Row className="layout-content">
       <Helmet>
         <title>{params.id ? EDIT_LOCATION_UNIT_GROUP : ADD_LOCATION_UNIT_GROUP}</title>
```

Comment:
should the title change too?

Reply:
Only change in header title not for tab title,

Reply:
@KipSigei what do you think?

Reply:
Yes, OpenSRP is adopting the name **Location unit group** in place of the  **Location Tags**

Reply:
The question is whether the document.title should be the same as the header or if its okay to leave it as ` Location unit group` while the page header is ` Location unit group | <group.name>`.

Reply:
The document title doesn't need to have the group name. 
As long as the page header has the group name then the user can easily know where they are.

Reply:
cool, thanks.